vin/test/fixtures/addresses.yml

```diff
@@ -1,13 +0,0 @@
-# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
-
-one:
-  street: MyString
-  city: MyString
-  state: MyString
-  zip: 1
```

Comment:
I have to dig into the test folder, i didn't wrote that, completely automated by creating the models through the rails CLI :)